src/js/popup.js

```diff
@@ -438,6 +438,7 @@ function refreshPopup() {
     $("#activate_site_btn").show();
     $("#deactivate_site_btn").hide();
     $("#disabled-site-message").show();
+    $("#title").toggleClass("disabled_site_popup");
```

Comment:
Some nits:
- CSS identifiers should be dash-separated (going forward): `disabled-site-popup`
- Maybe calling this class something like "faded" is more to the point?
- Should probably pass `true` as the second argument to `toggleClass()` to be clear that we want to add the class, not remove it. Hmm, we should just use `addClass()` then, right?

Reply:
good call. did points 1 & 3 in the most recent commit, expanded on point 2 to rename the css class `.faded-bw-color-scheme` to make it more specific